src/test/ui/return/return-impl-maybe-sized.stderr

```diff
@@ -0,0 +1,7 @@
+error: return type should be sized
+ --> src/test/ui/return/return-impl-maybe-sized.rs:1:17
+  |
+1 | fn test_fn() -> impl ?Sized {}
```

Comment:
The output doesn't seem to be updated (notice the line number), needs to run the test with `--bless` again, I think.

Reply:
thx, updated, and i spend some time to figure out how the test works